src/v/random/generators.h

```diff
@@ -35,6 +35,9 @@ static thread_local std::default_random_engine gen(internal::get_seed());
 bytes get_bytes(size_t n = 128 * 1024);
 ss::sstring gen_alphanum_string(size_t n);
 
+static constexpr size_t alphanum_max_distinct_strlen = 32;
+ss::sstring gen_alphanum_max_distinct(size_t max_cardinality);
```

Comment:
please write a comment here about what this is doing.

Reply:
Covered in another comment and addressed. PR update is in progress--expect update soon